lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb

```diff
@@ -127,81 +265,162 @@ def self.bump_version_release
           return verified_version
         end
 
+        # Update the build.gradle file with new versionName and versionCode values, both or the defaultConfig and vanilla flavors
```

Comment:
```suggestion
        # Update the build.gradle file with new versionName and versionCode values, both or the `defaultConfig` and `vanilla` flavors
```

In other places you've used code-quotes for `defaultConfig` –not sure if you want to stay consistent here? Same note for `vanilla`.

Reply:
… and for `build.gradle` probably 😜